src/Sulu/Bundle/AdminBundle/Resources/public/scss/modules/sticky-toolbar.scss

```diff
@@ -0,0 +1,31 @@
+.sticky-toolbar.fixed {
+    .list-toolbar-container {
+        width: 100%;
+        padding-right: 100px;
+        position: absolute;
+        background: #f1f1f1;
+        z-index: 98;
+        top: 30px;
+        padding-top: 10px;
+
+        &.fixed {
+            padding-right: 100px;
```

Comment:
What's this padding then for? It just adds some padding to the right, and doesn't change the appearance at all.